resources/Changelog.txt

```diff
@@ -289,8 +289,9 @@ v2.9:
     - Bump junit from 4.11 to 4.13.1 (@dependabot)
     - Fix syntax for "/temp..." commands (@l1ttleO)
     - Add timed subgroups (@l1ttleO, @ElgarL)
-    - Add PAPI support...
+    - Add PAPI support (@ElgarL)

```

Comment:
Lets not put a credit on everything we add to the Changelog. Credits on new Language additions or large features is fine else we are quickly going to spam names in what should be informational.